### ISSUE TITLE
Specify no BCC on /mailinglists

### ISSUE THREAD

**Issue by michaeljohansen** [dotKom/onlineweb4#994]
TL;DR: Add a sentence about not using BCC on /resourcecenter/mailinglists

In the interest of transparency, possibilties for follow-up and monitoring, I have consistently told people who use BCC on Online's stud.org. mailinglists to not do that. I'd love to have that in a sentence on the page itself so that I could reference that page more easily. Whenever BCC is used it's consitently external organizations (not stud.orgs.).

Why not just do it myself? In case anyone disagrees. Hope not. Should be a no-brainer unless I've totally missed something. What do you think @melwil, @lizter, @hernil, @kradalby ?

**Comment by sardred** [comment]
I made a pull request for this issue now, but the specific placement in the text, and the wording, is open for change.
